lib/bibliothecary/parsers/conda.rb

```diff
@@ -10,18 +10,22 @@ def self.mapping
         {
           match_filename("environment.yml") => {
             parser: :parse_conda_manifest,
-            kind: "manifest"
+            kind: %w[manifest lockfile]
           },
           match_filename("environment.yaml") => {
             parser: :parse_conda_manifest,
-            kind: "manifest"
+            kind: %w[manifest lockfile]
           }
         }
       end
 
       def self.parse_conda_manifest(file_contents)
```

Comment:
I think this method is no longer used? Might be good to delete it and replace the `parser:` in the mapping with something explicitly no-op.

Reply:
Isn't `mapping.parser` still getting called from the CLI functions?

Reply:
addressed! making the new thing from keith's suggestion use the same stuff as the parser.